IETF-RFC.md

```diff
@@ -275,11 +275,20 @@ itself be an object containing the following fields:
 * OPTIONAL: capabilities (array of string) - The optional capabilities supported by this OCM Server.
           As implementations MUST accept Share Creation Notifications to be compliant,
           it is not necessary to expose that as a capability.
-          Example: `["/notifications"]`. The array MAY include for instance:
-    * `"/notifications"` - to indicate this OCM server is capable of processing OCM Notifications
-    * `"/invite-accepted"` - to indicate that this OCM server is capable of processing Invite Acceptance Requests.
-    * `"/mfa-capable"` - to indicate that this OCM server can apply a Sending Server's MFA requirements for a Share on their behalf.
-        
+          Example: `["receive-code", "webdav-uri"]`. The array MAY include for instance:
+    * `"enforce-mfa"` - to indicate that this OCM server can apply a Sending Server's MFA requirements for a Share on their behalf.
+    * `"webdav-uri"` - to indicate that this OCM server can append a relative URI to the path listed for WebDAV in the appropriate `resourceTypes` entry
+    * `"receive-code"` - to indicate that this OCM server can receive a `code` as part of a Share Creation Notification, and exchange it for a bearer token at the Sending Server's `/token` API endpoint.
+* OPTIONAL: criteria (array of string) - The criteria for accepting a Share Creation Notification.
```

Comment:
Yes!

`A`'s system could check if it can satisfy all the criteria of `B`'s system, although of course with allowlist and denylist, it can never be sure. If it knows that the share will fail, it can make the UX nicer by not offering it as an option in the first place.

Another way to use this would be for debugging.

Reply:
Understood! Then it's probably good to include that in the spec?